cmd/notation/key.go

```diff
@@ -31,12 +36,21 @@ var (
 	keyAddCommand = &cli.Command{
 		Name:      "add",
 		Usage:     "Add key to signing key list",
-		ArgsUsage: "<key_path> <cert_path>",
+		ArgsUsage: "[<key_path> <cert_path>]",
 		Flags: []cli.Flag{
 			&cli.StringFlag{
 				Name:    "name",
 				Aliases: []string{"n"},
-				Usage:   "key name",
+				Usage:   "key name (required if --plugin is set)",
+			},
```

Comment:
The alpha 1 release supported signing only with a local key and cert file. For the newer `config.json` and CLI experience, key name (friendly name) should always be specified while adding a new key, as users will subsequently only refer to the key using its name for `notation sign` command.